src/apikey/routers/manage.js

```diff
@@ -79,24 +79,32 @@ router.get('/keys', checkAuth, async (req, res, next) => {
       return next({ message: `Cannot parse config [${config}]`, stackTrace: err });
     }
 
-    allKeys.push({ [key]: config });
+    allKeys.push({ apikey, config });
   }
 
+  const sortApikey = (a, b) => {
+    if (a.config.name < b.config.name) { return -1; }
+    if (a.config.name > b.config.name) { return 1; }
```

Comment:
Config could be `null`

```suggestion
    if (a.config?.name < b.config?.name) { return -1; }
    if (a.config?.name > b.config?.name) { return 1; }
```